infrastructure/server-setup/backups.yml

```diff
@@ -174,3 +174,25 @@
       when: "'backups' in groups and groups['backups'] | length > 0 and ssh_test.rc != 0"
       tags:
         - backups
+    
+    - name: Create a file to transfer to the backup server
+      shell: touch /tmp/test_file.txt
+      tags:
+        - backups
+
+    - name: Check ability to transfer a file to the backup server
+      shell: rsync -avz -e "ssh -o StrictHostKeyChecking=no -o ConnectTimeout=5" /tmp/test_file.txt {{ backup_server_user }}@{{ destination_server }}:{{ backup_server_remote_target_directory }}/test_file.txt
+      remote_user: '{{ crontab_user }}'
+      register: rsync_test
+      when: "'backups' in groups and groups['backups'] | length > 0"
+      tags:
+        - backups
+         
+    - name: Check if file can be transferred to the staging server from backup server
+      shell: ssh -o StrictHostKeyChecking=no -o ConnectTimeout=5 {{ backup_server_user }}@{{ destination_server }} 'rsync -avz -e "ssh -o StrictHostKeyChecking=no -o ConnectTimeout=5" {{ backup_server_remote_target_directory }}/test_file.txt {{ crontab_user }}@{{ inventory_hostname }}:/tmp/test_file.txt'
```

Comment:
This doesn't look correct. Instead of backup machine sending the file to the staging server, the staging server should be downloading it using rsync. This ansible step also needs to be restricted to be ran only for staging servers